app/presenters/idv/in_person/ready_to_verify_presenter.rb

```diff
@@ -27,15 +34,18 @@ def needs_proof_of_address?
         !enrollment.current_address_matches_id
       end
 
+      def service_provider
+        enrollment.service_provider
+      end
+
+      def sp_name
+        service_provider ? service_provider.friendly_name : ''
```

Comment:
I have a question about what should happen if there is no service provider. We'll see that a lot in lower environments today. And after we finish the pilot I think there will be ways for production users to get into the IPP flow without an associated partner, such as by going through the password reset flow

I left a [question](https://www.figma.com/file/QBk8EwtDVqcIYtII1BlJWI?node-id=152:264#295113616) on the Figma file about it

Reply:
that's a good question. i left it as an empty string and so in examples i looked at w/o one it looks like an unfinished sentence -it definitely is not ideal. would be good to adjust for sure.

Reply:
Lizzie suggested we discuss it at design critique on Monday so I'm going to plan to attend for that. Do you want to join too?

Reply:
We discussed and tentatively agreed to fall back to `Login.gov` when there's no service provider, similar to how we do in other places

```suggestion
        service_provider ? service_provider.friendly_name : APP_NAME
```

Reply:
this is being addressed in #7256